app/src/main/java/com/android/sample/MainActivity.kt

```diff
@@ -85,8 +87,48 @@ fun PlateSwipeApp() {
         startDestination = Screen.CREATE_RECIPE,
         route = Route.CREATE_RECIPE,
     ) {
-      composable(Screen.CREATE_RECIPE) { CreateRecipeScreen(navigationActions) }
+      composable(Screen.CREATE_RECIPE) { backStackEntry ->
```

Comment:
Could you please explain what you’re doing here? Is there a reason why the `CreateRecipeViewModel` can't be declared just once alongside the other ViewModels?

Thank you!

Reply:
Hey, you are right, we now declare it in the `MainActivity` with all the other `viewModels`